src/guides/v2.3/architecture/archi_perspectives/components/modules/mod_and_areas.md

```diff
@@ -50,7 +50,7 @@ You can enable or disable an area within a module. If this module is enabled, it
 
 Magento processes a URL request by first stripping off the base URL. The first path segment of the remaining URL identifies the request area.
 
-After the area name, the URI segment specifies the *frontname*. When an HTTP request arrives, Magento extracts the handle from the URL and interprets it as follows: `[frontName]/[controller folder]/[controller class]` where `frontName` is a value defined in the module. For example, in `catalog/product/view`, `catalog` is the (frontName defined in the module area's `routes.xml`)[{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/etc/frontend/routes.xml#L10], `product` is the (controller folder)[{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Controller/Product], and `view` is the (controller class)[{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Controller/Product/View.php]. For deeper directory structures, the controller folders are separated with `_` (for example, `catalog/product_compare/add` for `Magento/Catalog/Controller/Product/Compare/Add.php`).
+After the area name, the URI segment specifies the *frontname*. When an HTTP request arrives, Magento extracts the handle from the URL and interprets it as follows: `[frontName]/[controller folder]/[controller class]` where `frontName` is a value defined in the module. For example, in `catalog/product/view`, `catalog` is the [frontName]({{site.mage2bloburl}}/{{page.guide_version}}/app/code/Magento/Catalog/etc/frontend/routes.xml#L10) defined in the module area's `routes.xml`, `product` is the [controller folder]({{site.mage2bloburl}}/{{page.guide_version}}/app/code/Magento/Catalog/Controller/Product), and `view` is the [controller class]({{site.mage2bloburl}}/{{page.guide_version}}/app/code/Magento/Catalog/Controller/Product/View.php). For deeper directory structures, the controller folders are separated with `_` (for example, `catalog/product_compare/add` for `Magento/Catalog/Controller/Product/Compare/Add.php`).
```

Comment:
Hi @SikailoISM. Thank you for your collaboration. Please, preserve spaces between parenthesis and the contents
 
```markdown
{{site.mage2bloburl}} -> {{ site.mage2bloburl }}
```

Reply:
Hi @rogyar. There are problem with spaces: in preview changes while editing I see link text instead of clickable text if I use spaces. But without spaces everything works fine.

Reply:
Done, please check ;3